src/test/java/cat/udl/eps/softarch/academicrecruit/steps/CommitteeMemberStepDefs.java

```diff
@@ -61,4 +68,20 @@ public void iAssignARankToAUserWithUsername(String rank, String username) throws
                         .with(AuthenticationStepDefs.authenticate()))
                 .andDo(print());
```

Comment:
You already have the .andDo(print()) to debug your code so no need for the println